lms/djangoapps/instructor/tests/test_proctoring.py

```diff
@@ -62,14 +112,29 @@ def test_no_tab_non_global_staff(self):
         self.assertNotIn('Allowance Section', response.content)
 
     @patch.dict(settings.FEATURES, {'ENABLE_SPECIAL_EXAMS': False})
-    def test_no_tab_flag_unset(self):
+    @ddt.data(
+        (True, False),
+        (False, True)
+    )
+    @ddt.unpack
+    def test_no_tab_flag_unset(self, enable_proctored_exams, enable_timed_exams):
         """
-        Special Exams tab will not be visible if
-        the user is not a staff member.
+        Special Exams tab will not be visible if special exams settings are not enabled inspite of
+        proctored exams or timed exams is enabled
         """
+        self.setUpCourse(enable_proctored_exams, enable_timed_exams)
+
         self.instructor.is_staff = True
         self.instructor.save()
 
         response = self.client.get(self.url)
         self.assertNotIn(self.proctoring_link, response.content)
         self.assertNotIn('Allowance Section', response.content)
+
+    def _assert_proctoring_tab_is_available(self):
+        """
+        Asserts that proctoring tab is available for global and course staff
```

Comment:
`global and course staff` this should be updated, should be `available for logged in user.`